From dc824a79c86ca70c6c5bbd8eee847112d7d1b226 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 22 Feb 2024 10:22:27 +0100
Subject: [PATCH 0686/1695] dts: stm32: enable PKA on stm32mp257f-ev1

Sets PKA peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 826f8c0e548ded0dccd6a482e8481927555cc87e dts: stm32: enable PKA on stm32mp257f-ev1]
Change-Id: Iefef4c18fa171929e77b6ab62e258999685e9c8b
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362227
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 54b263ba9..cc7d6f6b8 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -354,6 +354,10 @@
 	status = "okay";
 };
 
+&pka {
+	status = "okay";
+};
+
 &pwr {
 	status = "okay";
 	wakeup-gpios = <&gpioa 0 GPIO_ACTIVE_HIGH>, <0>, <0>, <0>, <0>, <0>;
-- 
2.47.2

